### ISSUE TITLE
Improve community support

### ISSUE THREAD

**Issue by bjohansebas** [expressjs/discussions#258]
On the Express page, links to other platforms like [Gitter](https://gitter.im/expressjs/express), the [Google group](https://groups.google.com/group/express-js) or IRC channels are promoted, but these resources no longer seem to be used by the community. In the case of Gitter, the last message is spam for a Telegram group.

It would be good to consider other ways to help the community, such as setting up a Discord server, creating a community on X, etc.

**Comment by wesleytodd** [comment]
Those used to all be active, times have for sure changed. The problem we have is we never had enough contributors to run those. If we would like to make a program to help folks get involved in that I would be down for it.

**Comment by bjohansebas** [comment]
Given that many projects like ESLint, Vercel, and Biome.js use Discord, implementing a server could make it easier for people to connect and discuss Express-related topics. We could also add channels for blog ideas, announcements, releases, and support.

@wesleytodd If you like the idea, I could set up the server and show how it could look.

**Comment by wesleytodd** [comment]
My only concern with Discord is we already are heavily using the OpenJS Slack, and I know I have stopped participating in the discords because it is just too many chat apps to keep open all the time. If you have not seen the activity in the OpenJS slack it might be worth checking out first before setting up a discord.

**Comment by bjohansebas** [comment]
The problem with Slack is that it’s mostly used for internal organizational matters rather than for the community. The way I see it, Discord would be more focused on providing support and helping those who have issues with implementing Express, similar to how other platforms have done it.

**Comment by wesleytodd** [comment]
Oh I fully agree. Slack sucks for communities. I just mean that the problem is collaborator participation, and we have a sponsored and well managed slack instance *already*.

**Comment by ljharb** [comment]
Personally i vastly prefer slack to discord for communities; discord’s UX is awful.

**Comment by wesleytodd** [comment]
Yeah thats true as well, but the community moderation things in discord are much better.

**Comment by ljharb** [comment]
More fully featured for sure - in my experience “deactivate account” and “delete comment” have been sufficient moderation tools for slacks i manage tho :-)

**Comment by bjohansebas** [comment]
@ljharb Keep in mind that on Discord there will be more external people interested in the project and its community, whereas Slack feels more closed-off from my perspective.

**Comment by ljharb** [comment]
That's subjective and very much not my experience - Slack is just as easy if not easier to join than Discord - but certainly it's possible.

**Comment by wesleytodd** [comment]
One thing to be clear about here is the difference between an official and unofficial community. I assumed this conversation was about an official one. If that is the case we need folks in there doing moderation and generally keeping thing in line with the projects values. This means likely TC members are around. That is my worry about discord right now, despite me using it for gaming I dont have it installed on my work computer and generally have decided my time was better spent in slack because all of the current TC members and contributors have seemed comfortable with using the OpenJS slack for our real-time comms.

Alternatively, an unofficial channel wouldn't need this stuff. So it might achieve a better community, which would be awesome if that happens. We don't need central control or anything, but I do think that people come to those things to find value in talking with the maintainers, and if the maintainers are not there they tend to fail (as happened with the previous attempts).

Anyway, I am pretty open to either as I do actively use both, but we should be clear on the goals.

**Comment by LeaderSix** [comment]
@bjohansebas +1 for bringing this up.

However, my vote goes to GitHub discussion. Perhaps we could just change our doc to post in our repo discussion for support. It could make more sense in terms of _TC involvement_. 

It will be much easier to reference issues, prs and possibly convert a discussion to an issue if needed. Seems cleaner to me. I suggest we promote GitHub discussion as our _official community_ and just see how this goes and grows. It will also not require extra moderation unlike Discord.

**Comment by bjohansebas** [comment]
@wesleytodd I agree, and what I wanted when starting this discussion is to have an official community on some platform for people who are interested in using this technology, like there used to be before. I also like @IamLizu idea; the important thing is to correctly select a platform where everyone feels comfortable, as well as properly informing users of these platforms.

@wesleytodd On the other hand, can I remove references to the mailing list, Gittler, and the IRC channel from the page?

**Comment by wesleytodd** [comment]
> On the other hand, can I remove references to the mailing list, Gittler, and the IRC channel from the page?

Oh yeah. Please do.

**Comment by bjohansebas** [comment]
I've seen that Node.js has an open proposal to have a Discord server. 

Ref: https://github.com/nodejs/admin/issues/872

**Comment by bjohansebas** [comment]
I've been thinking about this for a long time. I think it's best not to split support channels too much. Discord is great, but we're a small group, and none of us are full-time, so I no longer see Discord as viable.

Slack is great for internal communication among us, but not ideal for building a community there.

Maybe we can enable organization-level Discussions, and also enable Discussions in each repo that is a package

cc: @expressjs/express-tc @expressjs/triagers

**Comment by wesleytodd** [comment]
> but not ideal for building a community there.

I have been on the fence on this for a while, but I have found a TON of value in the OpenJS slack for building the community. Specifically it seems to have a much higher signal to noise ratio compared to other things. That said, I recognize that is not a goal in itself and there is value in "here is a place for everyone to go" is valuable.

> Maybe we can enable organization-level Discussions, and also enable Discussions in each repo that is a package

I am not sure how this would be different than what we have today. Can you help me understand? I thought the "real time comms" part was the more important thing missing, and GH discussions doesn't seem to help there does it?

**Comment by bjohansebas** [comment]
> I am not sure how this would be different than what we have today

Yeah, the important thing would be to respond as quickly as possible. Obviously, we can't guarantee we'll respond to all the issues, this is open source, and no one is paying us to maintain these libraries, so enabling a GitHub discussion at the organization level wouldn’t improve much.

Also, I think we should document Slack on the page https://expressjs.com/en/resources/community.html, since it's currently used for both the working groups and the development of Express itself. It's already in the footer, so it's not a bad idea to mention it there (cc @expressjs/docs-wg).
